src/hooks/login/useLedgerLogin.ts

```diff
@@ -129,6 +129,11 @@ export function useLedgerLogin({
 
     if (hasNativeAuth && !token) {
       token = await loginService.getNativeAuthLoginToken();
+      // fetchig block failed
+      if (!token) {
+        console.warn('Login cancelled.');
```

Comment:
Please see my comment above. I see that the log message is used for all login methods. If you decide to change the text, please take into consideration also the other places